File: factorials.cpp
//Factorial!
//https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/find-factorial/
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int n,fact=1;
	cin >> n;
	for( int i = 2 ; i <= n ; i++) {
		fact = fact * i ;
		
	}
	cout<<fact;
	
	return 0;
}

File: factorials.py
//Factorial! 
//https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/find-factorial/
def fact():
    f =1
    for i in range(1,n+1): 
        f = f * i
    return f
n = int(input())
print(fact())
